src/controllers/dynakube/activegate/capability/capability.go

```diff
@@ -114,8 +115,8 @@ func (c *capabilityBase) Volumes() []corev1.Volume {
 	return c.volumes
 }
 
-func CalculateStatefulSetName(capability Capability, instanceName string) string {
-	return instanceName + "-" + capability.ShortName()
+func CalculateStatefulSetName(capability Capability, dynakubeName string) string {
+	return dynakubeName + "-" + capability.ShortName()
 }
 
 // Deprecated
```

Comment:
Nitpick: The deprecated comments could be annotated with what should be used instead